### PR TITLE
upload images and modify members

### PR DESCRIPTION
**Pull request by yuchen0816**

### CODE REVIEW

File: img/chenchen.jpg
Comment (dspp779) [inline img/chenchen.jpg:1]: size too big and too high

File: js/members.js [hunk 1]
```diff
@@ -114,6 +114,138 @@ const members = {
...
+            img: 'img/chien.jpg',
+            color: 'LightYellow',
+            intro: 'Para la vida que quiero, no hay atajos.',
+            link: '',
+        },
+        {
+            name: '許承恩',
+            alias: 'Irene',
+            research: '#',
+            img: 'img/irene.jpg',
```
Comment (dspp779) [inline js/members.js]: `irene123` not `irene`

File: js/members.js [hunk 2]
```diff
@@ -114,6 +114,138 @@ const members = {
         },
     ],
     'ms-students': [
+        {
+            name: '江詠筑',
+            alias: 'Angie',
+            research: '#',
+            img: 'img/chuchu.jpg',
```
Comment (dspp779) [inline js/members.js:121]: it's `.jpeg` for the actual file.

File: img/irene123.jpg
Comment (dspp779) [inline img/irene123.jpg:1]: size too big

File: js/members.js [hunk 3]
```diff
@@ -213,84 +345,6 @@ const members = {
             intro: '^•ﻌ•^ฅ',
             link: '',
         },
-    ],
```
Comment (dspp779) [inline js/members.js:207]: should not delete old people🥲

**Review by dspp779 — CHANGES_REQUESTED** [review]

File: js/members.js (hunk 2 above)
Comment (yuchen0816) [reply js/members.js:121]: change the file name to jpg

File: img/chenchen.jpg
Comment (yuchen0816) [reply img/chenchen.jpg:1]: update smaller file

File: js/members.js (hunk 1 above)
Comment (yuchen0816) [reply js/members.js]: irene123

File: img/irene123.jpg
Comment (yuchen0816) [reply img/irene123.jpg:1]: update smaller size

File: js/members.js (hunk 3 above)
Comment (yuchen0816) [reply js/members.js:207]: put old people to alumni

**Review by dspp779 — APPROVED** [review]
LGTM

**Review by dspp779 — APPROVED** [review]

**Pull request merged**